Provider/src/EntityFramework.Firebird.Tests/QueryTests.cs

```diff
@@ -164,4 +224,12 @@ class Baz
 		public string BazString { get; set; }
 		public ICollection<Foo> Foos { get; set; }
 	}
+	class Qux
```

Comment:
Does this name has special meaning? Could something simple and readable be used?

Reply:
https://en.wikipedia.org/wiki/Metasyntactic_variable#General_usage
Foo, Bar, and Baz are already taken... ;-)

Reply:
Ahh, missed that.